pathfinder/tests/fixtures/core.py

```diff
@@ -1,11 +1,65 @@
-from typing import List
+import random
+from typing import List, Dict, Tuple
 from unittest.mock import Mock
 
 import pytest
-
+from _pytest import monkeypatch
+from eth_utils import to_checksum_address, encode_hex, keccak
 from pathfinder.contract.token_network_contract import TokenNetworkContract
 from pathfinder.pathfinding_service import PathfindingService
 from pathfinder.token_network import TokenNetwork
+from pathfinder.utils.types import ChannelId, Address
+
+
+@pytest.fixture
+def populate_token_networks(
+    token_networks: List[TokenNetwork],
+    token_network_contracts: List[TokenNetworkContract],
+    monkeypatch: monkeypatch
+) -> None:
+
+    def get_channel_deposits_patched(
+        self: TokenNetworkContract,
+        channel_id: ChannelId
+    )-> Dict[Address, int]:
+
+        random.seed(channel_id)
+        deposit1 = random.randint(0, 1000)
+        deposit2 = random.randint(0, 1000)
+        participant1, participant2 = self.get_channel_participants(channel_id)
+        return {participant1: deposit1, participant2: deposit2}
+
+    def get_channel_participants_patched(
+        self: TokenNetworkContract,
+        channel_id: ChannelId
+    ) -> Tuple[Address, Address]:
+        random.seed(channel_id)
```

Comment:
We should probably switch to generating reusable private keys here before merging this PR, instead of generating addresses directly, so we can fake signatures for balance proofs and fee updates later.